specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json

```diff
@@ -0,0 +1,599 @@
+      "properties": {
+        "value": {
+          "type": "object",
+          "description": "Value for producer record"
+        },
+        "key": {
+          "type": "object",
+          "description": "Key for producer record"
+        },
+        "partition": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Partition ID for producer record"
+        }
+      },
+      "description": "A producer record"
+    },
+    "RESTBrokerResponse": {
+      "type": "object",
+      "properties": {
+        "brokers": {
+          "type": "array",
+          "description": "List of Kafka broker endpoints",
+          "items": {
+            "$ref": "#/definitions/RestProxyBrokerEndpoint"
+          }
+        }
+      },
+      "description": "List of Kafka broker endpoints"
+    },
+    "RESTPartitionMetadataResponse": {
+      "type": "object",
+      "properties": {
+        "partition_id": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Partition ID"
+        },
+        "earliest_offset": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Earliest offset for the topic-partition"
+        },
+        "latest_offset": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Latest offset for the topic-partition"
+        },
+        "leader": {
+          "description": "Leader broker endpoint for the topic-partition",
+          "$ref": "#/definitions/RestProxyBrokerEndpoint"
+        },
+        "replicas": {
+          "type": "array",
+          "description": "List of replica broker endpoints for the topic-partition",
+          "items": {
+            "$ref": "#/definitions/RestProxyBrokerEndpoint"
+          }
+        },
+        "isr": {
+          "type": "array",
+          "description": "List of broker endpoints in ISR for the topic-partition",
+          "items": {
+            "$ref": "#/definitions/RestProxyBrokerEndpoint"
+          }
+        }
+      },
+      "description": "Metadata information about a topic-partition"
+    },
+    "RESTTopicListResponse": {
+      "type": "object",
+      "properties": {
+        "topics": {
+          "type": "array",
+          "description": "List of topic names",
+          "uniqueItems": true,
+          "items": {
+            "type": "string"
+          }
+        }
+      },
+      "description": "List of topics"
+    },
+    "RESTTopicMetadaResponse": {
+      "type": "object",
+      "properties": {
+        "topic": {
+          "type": "string",
+          "description": "Name of the topic"
+        },
+        "partitions": {
+          "type": "array",
+          "description": "List of partition metadata",
+          "items": {
+            "$ref": "#/definitions/RESTPartitionMetadataResponse"
+          }
+        }
+      },
+      "description": "Metadata about a specific topic"
+    },
+    "RestProxyBrokerEndpoint": {
```

Comment:
`RestProxyBrokerEndpoint` is named differently than all the rest. For example:  `RESTTopicMetadaResponse`. Note the caps for REST.

Reply:
fixed